#19 - add feed / blog listing

tbxsaugat opened the pull request:
This PR will will allow authors to generate a blog listing/feed using the query-index. Authors can customize limit and featured-card.

Fix #19 

Test URLs:
- Before: https://main--creditacceptance--aemsites.aem.page/drafts/smalla/feed
- After: https://smalla-feed--creditacceptance--aemsites.aem.page/drafts/smalla/feed 

Testing criteria - https://www.creditacceptance.com/car-buyers/express-lane

kailasnadh790 commented:
@tbxsaugat it doesn't seem to pick right image :
![image](https://github.com/user-attachments/assets/71c7e20e-ddb6-4160-b339-2122424099aa)
 vs
 
![image](https://github.com/user-attachments/assets/25c6e5c2-6a31-4474-8d98-977c04320426)

kailasnadh790 commented:
how do we disable date ?

kailasnadh790 commented:
Cards are smaller on the live site.

kailasnadh790 commented:
check the "read" button behaviour against the live site.

kailasnadh790 commented:
Category tabs are wider on live site.

tbxsaugat commented:
@kailasnadh790 I have addressed your comments and it's ready to review. Regarding the images, I noticed that all the images in the query-index sheet have the same aspect ratio and are quite similar. Could you look into this?

ryanmparrish reviewed (approved):
Looks good. I only see one style thing to touch on. In the live site, there is a card hover drop shadow effect. Might want to just add that to these is all.

kailasnadh790 commented:
@tbxsaugat thanks !!
